cmd/analyzer/analyzer.go

```diff
@@ -85,7 +85,7 @@ func parseFlags(version string) cliFlags {
 	fs.StringVar(&cli.viewsOpts.BigQueryCredentialsJSON, "bigquery-credentials-json", "", "Google Cloud service account credentials JSON with access to BigQuery")
 	fs.StringVar(&cli.viewsOpts.ViewershipEventsTable, "viewership-events-table", "livepeer-analytics.viewership.staging_viewership_events", "BigQuery table to read viewership events from")
 	fs.StringVar(&cli.viewsOpts.ViewershipSummaryTable, "viewership-summary-table", "livepeer-analytics.viewership.staging_viewership_summary_by_video", "BigQuery table to read viewership summarized metrics from")
```

Comment:
While at it, would you mind fixing these too? These should be:
```suggestion
	fs.StringVar(&cli.viewsOpts.ViewershipEventsTable, "viewership-events-table", "livepeer-analytics.staging.viewership_events", "BigQuery table to read viewership events from")
	fs.StringVar(&cli.viewsOpts.ViewershipSummaryTable, "viewership-summary-table", "livepeer-analytics.staging.viewership_summary_by_video", "BigQuery table to read viewership summarized metrics from")
```

Reply:
Actually nevermind I confused myself, the current values are correct. Deleting comment :loading: